ui/src/app/dashboard/dashboard.module.ts

```diff
@@ -21,7 +21,7 @@ import {ClrIconModule, ClrTooltipModule} from "@clr/angular";
   declarations: [
     DashboardComponent,
     TotalSummaryComponent,
-    GroupedSummaryComponent
+    GroupedSummaryComponent,
```

Comment:
revert, no need to touch this file in the PR

Reply:
Fixed.